### PR TITLE
Soar in for shifters (pattern-like skill for hawk form)

### PR DESCRIPTION
**Pull request by nerble**
- Soar in (pattern-like skill for hawk form), tested it a bunch, hopefully nothing major wrong.  Let me know if any problems with chosen locations; easy enough to change.
- A few spelling fixes
- Update to MOTD and Changes to reflect new skill added.

### CODE REVIEW

File: server/src/sft.c [hunk 1]
```diff
@@ -136,6 +136,280 @@ void do_spy (CHAR_DATA *ch, char *argument)
...
+bool is_valid_soar (CHAR_DATA *ch, int soar_index )
+{
+        /* Figures out which soar locations a character has access to. */
+
+        if (soar_index > MAX_SOAR || soar_index < 1)
+        {
+                return 0;
+        }
+
+        if ( soar_index == 1
```
Comment (fromage-fraser) [inline server/src/sft.c:339]: You might be able to get all of this stuff into a data structure, a list of which you can then iterate through.
```
[
    {
        soar_id => 1,
        min_level => 1,
        min_skill_form_hawk => 1,
        min_skill_soar => 1,
    },
    {
        soar_id => 2,
        min_level => 5,
        min_skill_form_hawk => 30,
        min_skill_soar => 25,
    },
    ...
]
```

**Review by fromage-fraser — APPROVED** [review]

File: server/src/sft.c (hunk 1 above)
Comment (shadenz1) [reply server/src/sft.c:339]: Yeah I agree with that. 

Or you could loop over with say multiples of 10 for the skill (or something like that)

File: server/src/sft.c [hunk 2]
```diff
@@ -136,6 +136,280 @@ void do_spy (CHAR_DATA *ch, char *argument)
...
+                                sprintf(tmp, "%d. %s\n", i, location->name);
+                                strcat(buf, tmp);
+                        }
+                }
+
+                send_to_char(buf, ch);
+                return;
+        }
+
+        if (!str_cmp(arg, "mem"))
```
Comment (shadenz1) [inline server/src/sft.c:190]: This is fantastic - allowing characters quick access back to some part of the leveling area seems like a great utility skill and makes Hawk form actually worthwhile

File: server/src/sft.c (hunk 1 above)
Comment (nerble) [reply server/src/sft.c:339]: Yeah honestly I thought about it, but iterating over a structure felt like overkill for such a minor/specific application which was unlikely to ever change.  Of course no great merit to the branching ugliness either. :)  Will put through for now and add a note to make one later.  Consider this TEMPORARY CODE. **laughs in I-want-to-believe-I'm-not-lying**

File: server/src/sft.c (hunk 2 above)
Comment (nerble) [reply server/src/sft.c:190]: Yeah I think I'll use it.  I've tried to make the ordinary soar spots places that give reasonable access to areas, too, and have them roughly increase in connected area level as you gain access to them.

**Pull request merged**